### PR TITLE
Development

### PR DESCRIPTION
**Pull request by cbatres522**
Please review and approve my hungry message

### CODE REVIEW

**Comment by BRabbit5** [comment]
Tbh it was pretty good...not better than snoopy..but pretty good. Grade: D-

**Comment by hananhanouna** [comment]
good

File: src/team/g1/DemoG1.java [hunk 1]
```diff
@@ -11,12 +11,12 @@ public static void main(String[] args) {
 
 		System.out.println("Started from the bottom now we here");
 
-		System.out.println("RIP to ma' Popeyes!");
-		
 		System.out.println(" Hanan");
 
 		// User Hanan
 		System.out.println(" we are having fun with github ");
+		System.out.println("I was hungry so I ate a snack");
```
Comment (Ctrlv-Package) [inline src/team/g1/DemoG1.java:18]: I wanted a snack too. anyway, i guess I'll approve it even though you didn't share.

Comment (BRabbit5) [inline src/team/g1/DemoG1.java:18]: No share means you a meanie

**Pull request merged**